packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.ts

```diff
@@ -23,7 +23,7 @@ export const serviceWalkthrough = async (
 ): Promise<Record<string, unknown>> => {
   const { inputs } = serviceMetadata;
   const { amplify } = context;
-  const { parseInputs } = await import(`${__dirname}/../question-factories/core-questions.js`);
+  const { parseInputs } = await import(`${__dirname}/../question-factories/core-questions.ts`);
```

Comment:
we can path.join here

Reply:
had to revert this line, as it's actually js in the lib directory